releases/1.6/edge/kubeflow-lite/bundle.yaml

```
@@ -0,0 +1,164 @@
+# Modified from https://github.com/canonical/kubeflow-ci/blob/main/releases/1.4/kubeflow-bundle.yaml
+
+bundle: kubernetes
+name: kubeflow-lite
+applications:
+  admission-webhook:
+    charm: admission-webhook
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: admission-webhook-operator
+  argo-controller:
+    charm: argo-controller
+    channel: 3.3/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  dex-auth:
+    charm: dex-auth
+    channel: 2.31/edge
+    scale: 1
+    trust: true
+    _github_repo_name: dex-auth-operator
+  envoy:
+    charm: envoy
+    channel: 1.20/edge
+    scale: 1
+    _github_repo_name: envoy-operator
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+  istio-pilot:
+    charm: istio-pilot
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+    options:
+      default-gateway: kubeflow-gateway
+  jupyter-controller:
+    charm: jupyter-controller
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: notebook-operators
+  jupyter-ui:
+    charm: jupyter-ui
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: notebook-operators
+  kfp-api:
+    charm: kfp-api
+    channel: 2.0/edge
+    scale: 1
+    _github_repo_name: kfp-operators
+  kfp-db:
+    charm: charmed-osm-mariadb-k8s
+    channel: latest/edge
```

This shouldn't change, but in past we've been pinning to latest/stable for all bundles.  
```suggestion
    channel: latest/stable
```